### ISSUE TITLE
Feature request: Service management

### ISSUE THREAD

**Issue by lorduskordus** [nokyan/resources#255]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the solution you'd like

Mission Center recently gained the ability to manage services (start, stop, restart, show details)

I'd love to see this implemented in Resources.

**Comment by nokyan** [comment]
Hi,
I don't really see why Resources would need this, I fear a services entry could further clutter up the sidebar.

**Comment by nokyan** [comment]
> From my experience as a desktop user, the situation around system monitoring apps was not very good compared to Windows before Mission Center and Resources popped up. I had to use multiple CLI tools to find out the temp of my GPU, whether a GPU is used for decoding and stuff like that.

Thank you. :)

> I believe a desktop user running a desktop environment should be able to control the system completely through GUI and that whenever you send the user to the command line, it is a sign of the desktop's unreadiness for casual non tech users.

I 100% agree with you on that, though I don't know how much a desktop user really interacts with systemd services. I personally rarely interact with them as they usually just work. On my home server on the other hand I've written a few services and of course also interacted with them.

> If you're not interested in this, it's okay, imo you are already greatly improving the Linux desktop.

Thanks. I'll keep the idea in my mind. :)

**Comment by nyabinary** [comment]
Yeah, I found this very useful in Mission Center. Being able to manage systemd services in general is a huge plus :P

**Comment by Journeycorner** [comment]
> A long time ago, I also searched for a GUI way to manage services but aside from one dead project, nothing existed.

@lorduskordus I have actually written a GUI that does exactly that: view and manage systemd units: https://github.com/Journeycorner/systemd-gtk.

@nokyan Resources was the inspiration for creating it, so I would be happy to help integrate it into Resources!

 The reason I made it in the first place is that I believe units are at least as important to manage as processes and threads. For example, when I set up Arch Linux, I had to switch to the terminal to enable the Bluetooth service. I think users should not need to use the terminal if it can be avoided.

**Comment by miguilimzero** [comment]
That would be an awesome feature!
